climada/entity/exposures/litpop/gpw_population.py

```diff
@@ -0,0 +1,164 @@
+# FILENAME_GPW1 = '_30_sec.tif'
+YEARS_AVAILABLE = np.array([2000, 2005, 2010, 2015, 2020])
+# BUFFER_VAL = -340282306073709652508363335590014353408
+# Hard coded value which is used for NANs in original GPW data
+
+
+def load_gpw_pop_shape(geometry, reference_year, gpw_version=11, data_dir=None,
+                       layer=0, verbatim=True): # TODO: manually tested but no tests exist yet
+    """Read gridded population data from GPW TIFF
+    and crop to given shape(s).
+    
+    Parameters
+    ----------
+    geometry : shape(s) to crop data to in degree lon/lat.
+        for example shapely.geometry.(Multi)Polygon or shapefile.Shape
+        from polygon(s) defined in a (country) shapefile.
+    reference_year : int
+        target year for data extraction
+    gpw_version : int (optional)
+        Version number of GPW population data.
+        The default is 11
+    data_dir : Path (optional)
+        Path to directory with GPW data.
+        The default is SYSTEM_DIR.
+    layer : int (optional)
+        relevant data layer in input TIFF file to return.
+        The default is 0 and should not be changed without understanding the
+        different data layers in the given TIFF file.
+    verbatim : bool (optional):
+        if False, output in LOGGER is surpressed. Default is True.
+
+    Returns
+    -------
+    pop_data : 2D numpy array
+        contains extracted population count data per grid point in shape
+        first dimension is lat, second dimension is lon.
+    meta : dict
+        contains meta data per array, including "transform" with 
+        meta data on coordinates.
+    global_transform : Affine instance
+        contains six numbers, providing transform info for global GWP grid.
+        global_transform is required for resampling on a globally consistent grid
+    """
+    # check whether GPW input file exists and get file path
+    file_path = get_gpw_file_path(gpw_version, reference_year, data_dir=data_dir,
+                                  verbatim=verbatim)
+
+    # open TIFF and extract cropped data from input file:
+    src = rasterio.open(file_path)
+    global_transform = src.transform
+    pop_data, out_transform = rasterio.mask.mask(src, [geometry], crop=True,
+                                                  nodata=0)
+
+    # extract and update meta data for cropped data and close src:
+    meta = src.meta
+    meta.update({"driver": "GTiff",
+                 "height": pop_data.shape[1],
+                 "width": pop_data.shape[2],
+                 "transform": out_transform})
+    src.close()
+    return pop_data[layer,:,:], meta, global_transform
+
+def get_gpw_file_path(gpw_version, reference_year, data_dir=None, verbatim=True):
+    """Check available GPW population data versions and year closest to
+    reference_year and return full path to TIFF file.
+
+    Parameters
+    ----------
+    gpw_version : int (optional)
+        Version number of population data.
```

Comment:
```suggestion
    gpw_version : list(int), optional
        Version numbers of population data.
```

Reply:
Good point, I completely clean up the version control and just set one default in the CONFIG